mix.exs

```diff
@@ -84,4 +87,39 @@ defmodule UeberauthAuth0.Mixfile do
       links: %{GitHub: "https://github.com/achedeuzot/ueberauth_auth0"}
     ]
   end
+
+  defp aliases do
+    [
+      {:"test.all", [&run_tests/1, &run_tests_with_config_from/1]}
+    ]
+  end
+
+  defp run_tests(_) do
+    IO.puts("""
+    \n#
+    # Running tests with DEFAULT configurations.
+    #\n
+    """)
+
+    Mix.shell.cmd(
+      "mix test --color",
```

Comment:
When running the tests, I get the following warning:

```
You have configured application :exvcr in your configuration file,
but the application is not available.

This usually means one of:

  1. You have not added the application as a dependency in a mix.exs file.

  2. You are configuring an application that does not really exist.

Please ensure :exvcr exists or remove the configuration.
```

I think it's due to the `application` function that we use that is a legacy since Elixir < 1.4 which is not supported by the library anyway.

I think `application` should only contain the following `extra_applications`, and no `applications` key.
```
extra_applications: [:logger, :runtime_tools]
```

Reply:
That removed the warning! 👏🏽